<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperMegaBitCoinBot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startEarningButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start earning money!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apiKeyEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Your API key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secretKeyEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Your secret key</string>
    </property>
   </widget>
   <widget class="QWidget" name="statsView" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
